eph/leasedReceiver.go

```diff
@@ -64,7 +64,12 @@ func (lr *leasedReceiver) Run(ctx context.Context) error {
 		lr.periodicallyRenewLease(ctx)
 	}()
 
-	handle, err := lr.processor.client.Receive(ctx, partitionID, lr.processor.compositeHandlers(), eventhub.ReceiveWithEpoch(epoch))
+	opts := []eventhub.ReceiveOption{
+		eventhub.ReceiveWithEpoch(epoch),
+		eventhub.ReceiveWithConsumerGroup(lr.processor.consumerGroup),
```

Comment:
I believe this will set the consumer group on the receiver to be empty string, rather than the default of `$Default` via https://github.com/Azure/azure-event-hubs-go/blob/master/receiver.go#L40

I'd prefer the consumer group either not be set if it's empty string, that the optional `eventhub.ReceiveWithConsumerGroup` would handle empty string and replace it with the default, or that the initializer for `EventProcessorHost` would set the initial value for consumer group to be `eventhub. DefaultConsumerGroup`.

Reply:
Thanks for your feedback

You're totally right, I added a check that only sets the consumer group if it is not an empty string.